internal/pkg/aws/s3/s3.go

```diff
@@ -191,12 +197,24 @@ func (s *S3) isBucketExists(bucket string) (bool, error) {
 	return true, nil
 }
 
-func (s *S3) upload(bucket, key string, buf io.Reader) (string, error) {
-	resp, err := s.s3Manager.Upload(&s3manager.UploadInput{
+type uploadInputOption func(in *s3manager.UploadInput)
+
+func withACLBucketOwnerFullControl() func(in *s3manager.UploadInput)  {
```

Comment:
Since S3's recommendation is to have the bucket owner own every object (https://docs.aws.amazon.com/AmazonS3/latest/userguide/about-object-ownership.html) should we remove `withACLBucketOwnerFullControl` as a function and just inline this setting in `upload`?